src/app/search-bar/search-bar.component.scss

```diff
@@ -23,7 +25,7 @@
     border-radius: 6px;
     margin-left: 5px;
     border: 1px solid black;
-    background-color: rgba(242,234,118,1);
+    background-color: $green1;
```

Comment:
gdy nie wiedziałem jak nazwać kolorek to korzystałem z tego 
https://www.color-name.com/

Reply:
okej zapisuje stronke na przyszłośc